src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -49,7 +49,7 @@ public void RecordStoppedEvent(ApiEvent stoppedEvent)
         /// Csv expected format:
         ///      2|silent_successful_count|failed_requests|errors|platform_fields
         ///      failed_request is: api_id_1,correlation_id_1,api_id_2,correlation_id_2|error_1,error_2
-        ///      platform_fields: region_1,region_2
+        ///      platform_fields: region_1,region_source_1,region_2,region_source_2
```

Comment:
Isn't this a breaking change? How will you tell the different between messages without region source and messages with region source?

Reply:
Or if you do not want to introduce version, add the new fields at the end. Because the number of fields is variable, it will be hard to distinguish versions.

Reply:
Leaving it as is for now as per discussion offline. Will have to use some strategy to add more fields to platform fields in future.